azure-pipelines.yml

```diff
@@ -637,17 +637,25 @@ jobs:
   strategy:
     matrix:
       Red Hat Universal Base Image 8:
-        image: ubi8
-        tag: 8.4
+        dockerfile: ubi
+        image: registry.access.redhat.com/ubi8/ubi:8.4
         artifact: rpm-ubi8
+        python_package: python39
+      Red Hat Universal Base Image 9:
+        dockerfile: ubi
+        image: registry.access.redhat.com/ubi9/ubi:9.0.0
+        artifact: rpm-ubi9
+        python_package: python3.9
```

Comment:
`python39` in RHEL 8 becames `python3.9`. It's bad design for RHEL 9 to introduce this inconsistency.